src/celengine/category.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <set>
+#include <unordered_set>
+#include <map>
+#include "selection.h"
+
+class UserCategory
+{
+public:
+    struct Hasher
```

Comment:
i'm ok with the custom hasher, just a question - did it worked with a default one?

Reply:
Default didn't exist. That's why I had to write own.